src/app/shared/services/dialog.service.ts

```diff
@@ -15,7 +15,7 @@ export class DialogService {
     return this.dialog.open(component, config)
   }
 
-  public createConfirmDialog(title: string, message: string, okButtonText: string, onOk: () => void, severity?: DialogSeverity): void {
+  public createConfirmDialog(title: string, message: string, okButtonText: string, onOk: () => void, theme?: DialogColorTheme, severity?: DialogSeverity): void {
```

Comment:
I don't think we need to have `theme` and `severity` as optional properties. When opening a dialog, it is fair to assume that the user of the method knows what theme and severity they desire.

Reply:
This way we have default one, only down side of the is we can't set `severity ` without `theme`, when they are set it this way